src/levels/intro/branching.js

```diff
@@ -19,7 +19,7 @@ exports.level = {
     "vi": "Rẽ nhánh với Git",
     "sl_SI": "Branchanje v Gitu",
     "pl": "Rozgałęzienia w Gicie",
-    'it_IT': "Creare rami in Git",
+    "it_IT": "Creare rami in Git",
```

Comment:
super minor comment but usually discarding these changes are good so we don't introduce blame in the code that's not super accurate